android-project/app/src/main/java/oceantreasur/es/EndGameActivity.java

```diff
@@ -1,22 +1,37 @@
 package oceantreasur.es;
 
 import android.content.Intent;
+import android.content.res.Resources;
 import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
 import android.view.View;
 import android.view.animation.Animation;
 import android.view.animation.TranslateAnimation;
 import android.widget.ImageView;
+import android.widget.RelativeLayout;
 
 import com.bumptech.glide.Glide;
 
+import java.util.ArrayList;
+import java.util.Random;
+
 import oceantreasur.es.network.OceanTreasuresApplication;
 import oceantreasur.es.view.CustomButton;
 import oceantreasur.es.view.ViewUtils;
 
+import static oceantreasur.es.R.layout.fish;
+
 public class EndGameActivity extends BaseActivity {
 
+    private int MAX_FISH = 5;
+    private int MAX_DURATION = 14000;
+    private int MIN_DURATION = 8000;
+    private int MIN_TIME_OFFSET = 100;
+    private int MAX_TIME_OFFSET = 500;
     private ImageView image;
     private CustomButton button;
+    private ArrayList<View> fishArr = new ArrayList<>();
```

Comment:
Rename: don't include anything, regarding the variable types e.g. "fishes" or "fishViews". The plural form suggests that this is either an array or a collection.